client/java/src/main/java/com/linecorp/centraldogma/client/updater/CentralDogmaBeanFactory.java

```diff
@@ -44,24 +45,37 @@
  * Creates a new bean instance that mirrors its properties from Central Dogma.
  *
  * <p>In the following example, {@code mirroredFoo.getA()} and {@code getB()} will return the latest known
- * values retrieved from {@code /myProject/myRepo/foo.json} in Central Dogma:
+ * values retrieved from {@code /myProject/myRepo/foo.json} along with the latest revision in Central Dogma.
+ * If the latest values are not available yet, the revision will be set to null.
  *
  * <pre>{@code
- * > CentralDogmaFactory factory = new CentralDogmaFactory(dogma, new ObjectMapper());
- * > Foo mirroredFoo = factory.get(new Foo(), Foo.class);
- * >
- * > @CentralDogmaBean(project = "myProject",
- * >                   repository = "myRepo",
- * >                   path = "/foo.json")
- * > public class Foo {
- * >     private int a;
- * >     private String b;
- * >
- * >     public int getA() { return a; }
- * >     public void setA(int a) { this.a = a; }
- * >     public String getB() { return b; }
- * >     public void setB(String b) { this.b = b; }
- * > }
+ *  CentralDogmaFactory factory = new CentralDogmaFactory(dogma, new ObjectMapper());
+ *  Foo mirroredFoo = factory.get(new Foo(), Foo.class);
+ *
+ *  {@literal @}CentralDogmaBean(project = "myProject",
+ *                     repository = "myRepo",
+ *                     path = "/foo.json")
+ *  public class Foo {
+ *      private int a;
+ *      private String b;
+ *
+ *      public int getA() { return a; }
+ *      public void setA(int a) { this.a = a; }
+ *      public String getB() { return b; }
+ *      public void setB(String b) { this.b = b; }
+ *      public Revision getRevision() { return null; }
+ *  }
+ * }</pre>
+ *
+ * <p>In the following example, callback will be called when a property is updated with a new value:
+ *
+ * <pre>{@code
+ *  CentralDogmaFactory factory = new CentralDogmaFactory(dogma, new ObjectMapper());
+ *  Consumer<Foo> fooUpdatedListener = (Foo f) -> {
+ *     System.out.println("foo has updated to: " + f);
```

Comment:
three spaces here? or you can remove the space at the very front of lines(73, 74, 76).